src/components/Line/index.js

```diff
@@ -54,21 +51,19 @@ const Line = ({
   let circles = null;
   if (drawPoints) {
     const subDomain = xScale.domain().map(p => p.getTime());
-    circles = data
-      .filter(d => {
-        const x = xAccessor(d);
-        return x >= subDomain[0] && x <= subDomain[1];
-      })
-      .map(d => (
-        <circle
-          key={xAccessor(d)}
-          className="line-circle"
-          r={3}
-          cx={xScale(xAccessor(d))}
-          cy={boundedSeries(yScale(yAccessor(d)))}
-          fill={color}
-        />
-      ));
+    circles = (
+      <Points
+        data={data.filter(d => {
+          const x = xAccessor(d);
+          return x >= subDomain[0] && x <= subDomain[1];
+        })}
+        xAccessor={xAccessor}
+        yAccessor={yAccessor}
+        xScale={xScale}
+        yScale={yScale}
+        color={color}
```

Comment:
Possibly `opacity`, `opacityAccessor`, `strokeWidth`, `strokeWidthAccessor` could also be provided as props for Line -> Point (i.e. raw datapoints)

Reply:
yeah, but then it gets weird because you'd expect the line to change thickness/opacity/etc. I decided to leave this out until we have an answer for this strange API.